docs/API-specification.md

```diff
@@ -0,0 +1,160 @@
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+# API specification
+
+In order to be able to use yaml to define the objects required by apisix in k8s, the following structure is defined.
+
+## ApisixRoute
+
+`ApisixRoute` corresponds to the `route` object in apisix.
```

Comment:
That's not really right, you may try to compare the `Route` in APISIX and `ApisixRoute`.